ci/travis/detect-changes.py

```diff
@@ -0,0 +1,26 @@
+# Script used for checking changes for incremental testing cases
+
+import os
+import sys
+
+if __name__ == "__main__":
+
+    CHANGED_FILES = sys.argv[1]
+
+    files = CHANGED_FILES.split("\n")
+
+    ray_tune_tests = "FALSE"
+    ray_rllib_tests = "FALSE"
+    ray_tests = "FALSE"
+
+    for file in files:
```

Comment:
File is a reserved keyword in python, so I would change this to something like `for changed_file in files`

Reply:
Sorry, not a keyword, but a built-in function.